src/porepy/numerics/mixed_dim/assembler.py

```diff
@@ -156,7 +164,13 @@ def assemble_matrix_rhs(
 
                             # Assign values in global matrix
                             var_key_name = self._variable_term_key(term, row, col)
-                            matrix[var_key_name][ri, ci] += loc_A
+                            # Check if the current block is None or not, it could
```

Comment:
Please explain why it can be none (initialization)

Reply:
i just realized by running the tests